[#50] Add user follows/followers page

BraveNewWord opened the pull request:
# Description:
I have added a followers/follow of a user at /user/:username/follows
Using mock data, Mock data of follower and follows are added in arrays 
There is no "customizable header component" showing the user's username yet


### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](../CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project
- [ ] My code has been commented
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

BraveNewWord committed:
set up follows page and add one follower

BraveNewWord committed:
add MUI List follower and add more mock followers to show scrolling

BraveNewWord committed:
Add switching followers/follow tab and alt-mock data for the viewing follows state

BraveNewWord committed:
add mock-follows with pfps so it works

nzbasic commented on frontend/src/pages/follows/FollowsPageController.jsx:
```diff
@@ -0,0 +1,31 @@
+ */
+const FollowsPageController = () => {
+    // gets the id from the current url 
+    const { id } = useParams()
```
This will be undefined, because the route used is /user/:username/follows, so it should be:

```js
const { username } = useParams()
```

You can also remove the parseInt below

BraveNewWord replied:
Ok I changed this

BraveNewWord committed:
fix getting username issue on follows

BraveNewWord committed:
Secret variables updated

BraveNewWord committed:
Secret variables updated

BraveNewWord committed:
Secret variables updated to lower case

BraveNewWord committed:
Secret variables updated to lower case

nzbasic reviewed (approved):
✅ Page uses the correct route
✅ Can use a button to swap between followers and follows list
✅ Users are shown correctly
✅ Logic is split from the view 
✅ Mock data is used appropriately

aaronyaosong reviewed (approved):
LGTM

BraveNewWord committed:
Secret variables

BraveNewWord committed:
Secret variables

harristomy committed:
Removed use of secrets for DB login (#68)

Github does not allow secrets to be accessed by workflows triggered from PRs from a fork
